Formula/libmongocrypt.rb

```diff
@@ -1,8 +1,8 @@
 class Libmongocrypt < Formula
```

Comment:
Is this a necessary change for the 5.0 GA, in that libmongocrypt 1.2.1 must be available in order for 5.0 bits to work? Or is it more of a drive-by? If the latter, I'd prefer we did it as a separate commit.

Reply:
I don't really know, and I don't have a way of testing. I'll make it separate, though.

Reply:
I think #116 already does it. Maybe we should just merge that too? @kevinAlbs were you waiting on a particular reviewer on that?

Reply:
#116 omitted the update to `BUILD_VERSION`. If it is all the same, we can close #116 and proceed with this. Otherwise I can update #116 and request a reviewer.

Reply:
OK. @deriamis I'd say keep your edits for mongocrypt, but please land this as two separate commits, one to update mongocrypt, and another for the 5.0 GA. Then close #116.

Reply:
I still had this in my stash, so I checked it back out and added another commit to this PR.